parse.go

```diff
@@ -415,6 +418,16 @@ func cmdFromStruct(name string, dest path, t reflect.Type) (*command, error) {
 			spec.placeholder = strings.ToUpper(spec.field.Name)
 		}
 
+		noFormSpecs := emptyLongAndShort(kvPairs)
+
+		if spec.env == "" && noFormSpecs {
+			errs = append(errs, fmt.Sprintf("%s.%s: short arguments must be one character only",
+				t.Name(), field.Name))
+			return false
+		} else if spec.env != "" && noFormSpecs {
+			spec.envOnly = true
+		}
+
```

Comment:
I think this whole change can be avoided by dropping the `envOnly` var from the spec and checking instead for `if spec.short == "" && spec.long == ""`

Reply:
:heavy_check_mark: